website/package.json

```diff
@@ -15,7 +15,7 @@
   "main": "n/a",
   "scripts": {
     "start": "docusaurus start",
-    "build": "docusaurus build",
+    "build": "node --max_old_space_size=4096 node_modules/@docusaurus/core/bin/docusaurus build",
```

Comment:
@darshkpatel @slorber Question based on the curiosity - why this line is required?

Reply:
Basically while processing the Markdown files along with building the rest of the website, Node Crashes due to OOM Error since by default memory is limited to 1.4 GB, This increases the max memory to 4 GB.

Reply:
This would also mean we'd use Circle CI's `docker medium` executor to its full potential on the main website.

Reply:
@darshkpatel Can this tweak be applied on the Docusaurs side instead of RN website script?

Reply:
I'm not sure how, maybe @slorber can chime in about it

Reply:
I think that the heap fix can safely live in the deployment scripts 🙂

Reply:
This should not be in docusaurus core. It's really related to the site, its size and the memory its custom plugins take to run.

Generally you put this either on your CI env variables or in scripts like this, so it should be fine.